### PR TITLE
Create page for Starksheet

### PR DESCRIPTION
**Pull request by ClementWalter**

### CODE REVIEW

File: data/starksheet.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  "name": "Starksheet",
+  "description": "Like a spreadsheet, powered by the open web: each cell can call any contract using any on-chain data.",
+  "short_description": "The web3 spreadsheet",
+  "tags": [
+    "Onramp",
+    "DeFi",
+    "NFTs",
+    "Social",
+    "DAOs"
+  ], //Remove all that don't apply
```
Comment (ismaeldm95) [inline data/starksheet.json:11]:
```suggestion
   "tags": [
    "NFTs",
    "Social"
  ],
```

We need to remove the comment in order for it to work.  
Also, ideally we have max 2 or 3 tags

File: data/starksheet.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "name": "BasicCellRenderer",
+        "address": "0x048ee4bccae62f50ec929c757d1c53141ac3866e5ca67d5cff0edaa8e833f34b"
+      },
+      {
+        "name": "math",
+        "address": "0x021e46c69d5e116b2d7fce672cad513038ea9ad1d8a32050139201f6168cdf8c" 
+      }
+    ]
+  ],
+  "dotw": false, //leave 'false'
```
Comment (ismaeldm95) [inline data/starksheet.json:50]:
```suggestion
  "dotw": false,
```
We need to remove the comment for it to work

File: data/starksheet.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "discord": "https://discord.gg/BtuD5HguDX",
+    "github": "https://github.com/the-candy-shop/starksheet-monorepo",
+    "youtube": "",
+    "medium": "",
+    "mirror": ""
+  },
+  "twitterName": "starksheet",
+  "teamInfo": {
+    "founded": "2022-05-24T00:00:00.000Z >",
+    "anonymous": false // If at least one of the founders has a public profile with their real name linked to the project, set this to true.
```
Comment (ismaeldm95) [inline data/starksheet.json:65]:
```suggestion
    "anonymous": false
```
We need to remove the comment for it to work

File: data/starksheet.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    "telegram": "",
+    "discord": "https://discord.gg/BtuD5HguDX",
+    "github": "https://github.com/the-candy-shop/starksheet-monorepo",
+    "youtube": "",
+    "medium": "",
+    "mirror": ""
+  },
+  "twitterName": "starksheet",
+  "teamInfo": {
+    "founded": "2022-05-24T00:00:00.000Z >",
```
Comment (ismaeldm95) [inline data/starksheet.json:64]:
```suggestion
    "founded": "2022-05-24T00:00:00.000Z",
```
Fixing date format

File: data/starksheet.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  "description": "Like a spreadsheet, powered by the open web: each cell can call any contract using any on-chain data.",
+  "short_description": "The web3 spreadsheet",
+  "tags": [
+    "Onramp",
+    "DeFi",
+    "NFTs",
+    "Social",
+    "DAOs"
+  ], //Remove all that don't apply
+  "contracts": [
```
Comment (ismaeldm95) [inline data/starksheet.json:12]:
```suggestion
  "audits": [],
  "tokens": [],
  "contracts": [
```
We need empty arrays for audits and tokens for it to work properly

**Review by ismaeldm95 — COMMENTED** [review]
Here's how it looks when added to dappland with all the suggested changes:


Screenshot 2022-10-06 at 16 16 14


Screenshot 2022-10-06 at 16 18 11

**Comment by graemeblackwood** [comment]
Closing this as it will be implemented by #34

**Pull request closed without merging**